underpants.js

```diff
@@ -301,12 +539,28 @@ var _ = {};
 *   _.extend(data, {a:"two"}); -> data now equals {a:"two"}
 */
 
+// create the function, take in theoretically infinite arguments, all of which should be objects.
+// iterate over every other object, modifying object 1.
+    // put each property as a property of the object 1 copy.
+// return the copy.
+
+_.extend = (...objs) => {
+    for (let i = 1; i < objs.length; i++) {
+        for (let key in objs[i]) {
+            objs[0][key] = objs[i][key];
+        }
+    }
```

Comment:
Part of the goal of this sprint is to get you moving away from using for loops whenever possible. You should be able to use two `_.each` statements here instead.